daemon/cluster/convert/container.go

```diff
@@ -566,3 +569,32 @@ func ulimitsToGRPC(u []*units.Ulimit) []*swarmapi.ContainerSpec_Ulimit {
 
 	return ulimits
 }
+
+func tmpfsOptionsToGRPC(options [][]string) string {
+	// The shape of the swarmkit API that tmpfs options are a string. The shape
+	// of the docker API has them as a more structured array of arrays of
+	// strings. To smooth this over, we will marshall the array-of-arrays to
+	// json then pass that as the string.
+
+	// Marshalling json can create an error, but only in specific cases which
+	// are not relevant. We can ignore the possibility.
+	jsonBytes, _ := json.Marshal(options)
+	return string(jsonBytes)
+}
+
+func tmpfsOptionsFromGRPC(options string) [][]string {
+	// See tmpfsOptionsToGRPC for the reasoning. We undo what we did.
+	var unstring [][]string
+	// We can't return errors from here, so just don't ever pass anything that
+	// could result in an error.
+	//
+	// Duh.
+	//
+	// If there is something erroneous, then an empty return value will result,
+	// which should not be catastrophic. Because we control the data that is
+	// marshalled (in tmpfsOptionsToGRPC), we can more-or-less ensure that only
+	// valid data is unmarshalled here. If someone does something like muck
+	// with the GRPC API directly, then they get footgun, no apologies.
+	json.Unmarshal([]byte(options), &unstring)
```

Comment:
You have to explicitly discard the error to appease the linter.
```suggestion
	_ = json.Unmarshal([]byte(options), &unstring)
```